lib/datadog/core/crashtracking/component.rb

```diff
@@ -81,8 +84,42 @@ def stop
           logger.error("Failed to stop crash tracking: #{e.message}")
         end
 
+        def runtime_callback_registered?
+          return false if Datadog::Core::RUNTIME_STACKS_FAILURE
+          return false unless Datadog.const_defined?(:RuntimeStacks, false)
+
+          Datadog::RuntimeStacks._native_is_runtime_callback_registered
+        rescue => e
+          logger.debug("Runtime stack callback status check not available: #{e.message}")
+          false
+        end
```

Comment:
If this exists only for testing, I recommend moving to the testing code directly (e.g. the less code we declare "is for production", the better)